projects/ui/src/components/Analytics/Silo/index.tsx

```diff
@@ -9,15 +14,28 @@ import Seeds from '~/components/Analytics/Silo/Seeds';
 import DepositedAsset from '~/components/Analytics/Silo/DepositedAsset';
 import WarningIcon from '~/components/Common/Alert/WarningIcon';
 import APY from '~/components/Analytics/Silo/APY';
-
-// const SLUGS = ['deposited_bean','deposited_lp','deposited_urbean','deposited_urlp','stalk','seeds',];
 import { FC } from '~/types';
 
+const SLUGS = [
+  'deposited_bean',
+  'deposited_lp',
+  'deposited_urbean',
+  'deposited_urlp',
+  'stalk',
+  'seeds',
+  'bean_vAPY',
+  'lp_vAPY',
+];
+
 const SiloAnalytics: FC<{}> = () => {
-  const [tab, handleChangeTab] = useTabs();
+  const [tab, handleChangeTab] = useTabs(SLUGS, 'deposits');
```

Comment:
This tab refers to analytics about the Silo, so would suggest using `silo` instead of `deposits`